compiler/src/dmd/cxxfrontend.d

```diff
@@ -728,6 +729,18 @@ bool builtinTypeInfo(Type t)
     return dmd.typinf.builtinTypeInfo(t);
 }
 
+Type makeNakedAssociativeArray(TypeAArray t)
+{
+    import dmd.typinf;
+    return dmd.typinf.makeNakedAssociativeArray(t);
+}
+
+TypeInfoDeclaration getTypeInfoAssocArrayDeclaration(TypeAArray t, Scope* sc)
+{
+    import dmd.typinf;
+    return dmd.typinf.getTypeInfoAssocArrayDeclaration(t, sc);
+}
```

Comment:
So you need these 2 symbols for GDC, but don't wanna make them `extern(C++)` directly? [LDC doesn't need them.]